src/webots/nodes/WbCharger.cpp

```diff
@@ -104,7 +104,8 @@ void WbCharger::updateMaterialsAndLights(double batteryRatio) {
     WbPbrAppearance *appearance = dynamic_cast<WbPbrAppearance *>(visualElement->node);
     WbLight *light = dynamic_cast<WbLight *>(visualElement->node);
     const WbRgb color(cr, cg, cb);
-    assert(!WbRgb(cr, cg, cb).clampValuesIfNeeded());
+    const bool clampNeeded = WbRgb(cr, cg, cb).clampValuesIfNeeded();
+    assert(!clampNeeded);
```

Comment:
This broke the nightly builds, see https://github.com/cyberbotics/webots/actions/runs/4527046581/jobs/7972592389